src/main/java/de/dennisguse/opentracks/ui/markers/MarkerEditActivity.java

```diff
@@ -59,13 +62,16 @@ public class MarkerEditActivity extends AbstractActivity {
 
     public static final String EXTRA_TRACK_ID = "track_id";
     public static final String EXTRA_MARKER_ID = "marker_id";
+    public static final String EXTRA_LOCATION = "location";
+    public static final String EXTRA_IS_NEW_MARKER = "is_new_marker";
 
     private static final String CAMERA_PHOTO_URI_KEY = "camera_photo_uri_key";
 
     private static final String NEW_MARKER_ID = "new_marker_id";
 
     private static final String TAG = MarkerEditActivity.class.getSimpleName();
     private Track.Id trackId;
+    private Location location;
```

Comment:
We have an internal value object for location. 
Reason was that the android location requires lat/lng which is sometimes not present (e.g. only elevation via barometer).

Reply:
Where is that? Even TrackPoint is using the android Location.

Reply:
Wasn't there a link to a Position class or record I saw in the email?